ros/processor/pcp_generator.py

```diff
@@ -0,0 +1,16 @@
+from ros.processor import app
+from ros.lib.config import get_logger
+from ros.lib.config import ROS_PCP_PROCESSOR_PORT
+
+
+logging = get_logger(__name__)
+
+
+class PCPGenerator:
```

Comment:
As it is backend processor service, could you run http server to scrap metrics from this processor?
Similar to https://github.com/RedHatInsights/ros-backend/pull/506/files#diff-6f48879fb5d9b6c7702c5db83686f993f3fc502e881d30167dd873207967c756L49 
 IIRC, earlier main.py file was for entry point for running background processing.

Reply:
done